generators/form/templates/formComplex.js.ejs

```diff
@@ -1,6 +1,11 @@
 import _ from 'lodash/fp';
-import pensionsSchema from 'vets-json-schema/dist/21P-527EZ-schema.json';
-import burialsSchema from 'vets-json-schema/dist/21P-530-schema.json';
+
+// Example of an imported schema:
+import fullSchema from '../<%= formNumber %>-schema.json';
+// In a real app this would be imported from `vets-json-schema`:
+// import fullSchema from 'vets-json-schema/dist/<%= formNumber %>-schema.json';
+
+import commonDefinitions from 'vets-json-schema/dist/definitions.json';
```

Comment:
Let's add a similar comment here about how this would not be imported directly in a real app (as it would be referenced in the backend schema).